Cloud Application Manager/DR Readiness/Create Node Pair.md

```diff
@@ -0,0 +1,51 @@
+{{{
+  "title": "Create Node Pair",
+  "date": "09-26-2019",
+  "author": "Sharon Wang",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+**In this article:**
+
+* [Overview](#overview)
+* [Audience](#audience)
+* [Prerequisites](#prerequisites)
+* [Create a Node Pair](#create-node-pair)
+
+### Overview <a name="overview"></a>
+
+This article is meant to assist users of Cloud Application Manager willing to use Disaster Recovery feature, trying to create new Replication Node Pairs
+
+### Audience <a name="audience"></a>
+
+All users of Cloud Application Manager who wants to use Disaster Recovery feature.
+
+### Prerequisites <a name="prerequisites"></a>
+
+* An active *Cloud Application Manager* account
+* An existing *AWS* account configured in an [*AWS*](../Deploying Anywhere/using-your-aws-account.md) provider.
+* Both CLC and AWS providers are registered
+
+ 
+### Create a Node Pair <a name="create-node-pair"></a>
+1. Click on **Disaster Recovery** on the navigation bar, then click on the button **New** to create a new pair.
+![image](https://user-images.githubusercontent.com/20582531/65722771-357c3b00-e06a-11e9-9ffd-9b4386eb679c.png)
```

Comment:
Please, use meaningful alternate texts and relative path to images (in all referenced images on this PR)